src/pages/en/core-concepts/routing.md

```diff
@@ -145,9 +141,18 @@ Consider the following project:
 
 ```
 
-- `pages/post/create.astro` - Will match `/post/create`
-- `pages/post/[pid].astro` - Will match `/post/1`, `/post/abc`, etc. But not `/post/create`
-- `pages/post/[...slug].astro` - Will match `/post/1/2`, `/post/a/b/c`, etc. But not `/post/create`, `/post/1`, `/post/abc`
+Astro needs to know which route should be used to build the page. To do so, it sorts them according to the following rules:
+
+- Static routes without path parameters will take precedence over all other routes
+- Dynamic routes using named parameters take precedence over rest parameters
+- Rest parameters have the lowest priority
+- Ties are resolved alphabetically
```

Comment:
Given this is a priority list, would it make sense for it to be numbered:
```suggestion
1. Static routes without path parameters will take precedence over all other routes
2. Dynamic routes using named parameters take precedence over rest parameters
3. Rest parameters have the lowest priority
4. Ties are resolved alphabetically
```

Reply:
I like the numbering in general, but I'm not sure I see ties resolved alphabetically as a "4", since I guess this could happen at any stage?

I would be OK with or without numbering, and if numbered, then a slight preference for "Ties resolved" to be more of a note than a 4th step. But these are all weak preferences.

Otherwise, LGTM!

Reply:
You're right! Then I think maybe bullet points are ok. I don't want to break up these 4 things into 3 numbers and a note.